src/profile/index.html

```diff
@@ -19,10 +19,36 @@
     <title>Things To Do In Rexburg</title>
   </head>
   <body>
-    <header></header>
-    <body>
-      <div>X</div>
-    </body>
-    <footer></footer>
+    <header>
+    </header>
+    <div class="myProfile">
+      <h1>My Profile</h1>
+      <img src="../public/images/avatar.png" alt="Users avatar">
```

Comment:
Vite has a warning to use this routes. If you can make these changes, it will be great!
<img width="445" alt="Screenshot 2023-03-20 at 8 59 30 PM" src="https://user-images.githubusercontent.com/94401116/226508229-48585f91-3453-47d2-ae8f-82a8221c4644.png">

Reply:
I fixed this issue.